pkg/importer/util.go

```diff
@@ -38,10 +40,16 @@ func MoveFile(src, dst string) error {
 	return err
 }
 
-// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file
-func StreamDataToFile(dataReader io.Reader, filePath string) error {
+// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file or block device
+func StreamDataToFile(dataReader io.Reader, filePath, volumeMode string) error {
 	// Attempt to create the file with name filePath.  If it exists, fail.
-	outFile, err := os.OpenFile(filePath, os.O_CREATE|os.O_EXCL|os.O_WRONLY, os.ModePerm)
+	var outFile *os.File
+	var err error
+	if volumeMode == string(v1.PersistentVolumeBlock) {
+		outFile, err = os.OpenFile(filePath, os.O_EXCL|os.O_WRONLY, os.ModePerm)
```

Comment:
curious about the `O_EXCL` here.  Does it do something special for block devices?  Otherwise I think it is only meant to be used with `O_CREATE`.

Reply:
I know it's weird but for case of block PV with scratch space, when the O_CRAETE is there, it says that /scratchSpace/blockDevice already exists.Changed the code accordingly. let me know what you think..